rules/game_process.tex

```diff
@@ -75,6 +78,8 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee must tell the GameController operator to switch to \texttt{initial} state once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).
```

Comment:
We should use this change to formalize/define when human members of the playing teams need to leave the field/stop touching their robots. 

In the past it varied wildly from team to team how late before initial human members of the playing teams where touching the robots. It should be somewhat consistent across all teams for fairness.

We could introduce a requirement like "The head referee  announces setup time will be ending in 60 seconds. All human members of the playing teams must leave the field and stop interacting with participating robots manually and via network before initial."